rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -261,7 +256,6 @@ then
 fi
 
 %if 0%{?suse_version}
-a2enmod rewrite dav dav_fs ssl version wsgi
```

Comment:
we need those (except for wsgi)

Reply:
I think i was hacked because i only wanted to remove wsgi !! thanks !!